src/tools/kdb/metals.cpp

```diff
@@ -13,11 +13,15 @@ MetaLsCommand::MetaLsCommand()
 
 int MetaLsCommand::execute (Cmdline const& cl)
 {
+
+	if(cl.arguments.size() != 1){
+		throw invalid_argument("1 argument required");
+	}
+
 	Key root (cl.arguments[0], KEY_END);
 	if (!root.isValid())
 	{
-		cerr << "Not a valid name supplied" << endl;
-		return 1;
```

Comment:
The documentation doesn't currently mention the return value of 1. Is this used anywhere? If not this new solution is better, right?